src/main/webapp/pages/details.html

```diff
@@ -1,81 +1,95 @@
+                </div>
             </div>
         </div>
-    </div>
 
-    <!-- suggestions for this idea -->
-    <div id="shadowBox" class="row">
-        <div id="likeDiv" class="container">
-            <p id="p" translate="COMMENTS"></p>
-        </div>
-        <div id="scrollCommentsDiv" class="scrollable container" scrollable-list update-function="detailsCtrl.loadPageForComments()">
-	        <div id="commentsDiv" class="container">
-	            <div class="withoutComments" translate="WITHOUT_COMMENTS" ng-show="detailsCtrl.comments == null || detailsCtrl.comments.length == 0"></div>
-	            <div class="media comment" ng-repeat="comment in detailsCtrl.comments" show-comment comment="comment" is-authenticated="authenticated">
-	            </div>
-	        </div>
-        </div>
+        <!-- suggestions for this idea -->
+        <div id="shadowBox" class="row">
+            <div id="likeDiv" class="container">
+                <p id="p" translate="COMMENTS"></p>
+            </div>
+            <div id="scrollCommentsDiv" class="scrollable container" scrollable-list update-function="detailsCtrl.loadPageForComments()">
+                <div id="commentsDiv" class="container">
+                    <div class="withoutComments" translate="WITHOUT_COMMENTS" ng-show="detailsCtrl.comments == null || detailsCtrl.comments.length == 0"></div>
+                    <div class="media comment" ng-repeat="comment in detailsCtrl.comments" show-comment comment="comment" is-authenticated="authenticated">
+                    </div>
+                </div>
+            </div>
+
+            <div class="container" ng-show="authenticated">
+                <form name="addCommentForm" class="addCommentForm" role="form" novalidate>
+                    <textarea id="newComment" placeholder="{{ 'ENTER_COMMENT' | translate }}" required ng-model="commentBody" style="height: 103px;"></textarea>
+                    <button id="btnAddComment" class="btn btn-success green" type="submit" ng-disabled="addCommentForm.$invalid" ng-click="addComment()"><i class="fa fa-share"></i>{{ 'ADD' | translate }}</button>
+                </form>
+            </div>
 
-        <div class="container" ng-show="authenticated">
-	        <form name="addCommentForm" class="addCommentForm" role="form" novalidate>
-				<textarea id="newComment" placeholder="{{ 'ENTER_COMMENT' | translate }}" required ng-model="commentBody" style="height: 103px;"></textarea>
-				<button id="btnAddComment" class="btn btn-success green" type="submit" ng-disabled="addCommentForm.$invalid" ng-click="addComment()"><i class="fa fa-share"></i>{{ 'ADD' | translate }}</button>
-			</form>
         </div>
 
     </div>
-
 </div>
-</div>
\ No newline at end of file
+<script type="text/ng-template" id="myModalContent.html">
```

Comment:
May be, it worth to move template in separate file:
<script type="text/ng-template" id="myModalContent.html">
		<jsp:include page="/WEB-INF/jsp/<path>.jsp"/>
	</script>